### PR TITLE
#184948242 Setup of unit testing in client

### PR DESCRIPTION
**Pull request by Mystica52**
**#184948242 Setup the unit test using jest on client**

**What does this PR do?**
have all setup of unit testing

**Description of Task to be completed?**
after cloning it and then 

- run npm init -y
- run "npm install jest react-script @testing-library/react @testing-library/jest-dom react-test-renderer
- run "npm install chai -- save-dev" for installation of chai
- then make sure that script for test in packege.json is react-script

How should this be manually tested?
for testing, use terminal and run "npm run test"

**Any background context you want to provide?**
there is implementation made in vs code with 3 folder src and components where there is app.js and tests folder where app.test.js include
**What are the relevant pivotal tracker/Trello stories?**
Story ID: [#184948242](https://www.pivotaltracker.com/story/show/184948242)
**screenshoot**
image
image

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+          command: npm ci
+          name: Install Packages
+      - run:
+          command: npm test
+          name: Run tests
+
+workflows:
+  test_app:
+    jobs:
+      - test
```
Comment (Emile-SHUMBUSHO) [inline .circleci/config.yml]: Remove an empty line from your codes

**Review by Emile-SHUMBUSHO — CHANGES_REQUESTED** [review]

**Review by khevin2 — APPROVED** [review]

**Review by AbdulKhaliq59 — APPROVED** [review]

**Review by AbdulKhaliq59 — APPROVED** [review]
Rebase  unless it LGTM

**Review by GasanaJr — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "react-scripts test  --coverage --watchAll --timeout=10000",
     "dev": "webpack-dev-server --mode development",
```
Comment (GasanaJr) [inline package.json:18]: add --exit to exit the testing after all tests are done

**Review by GasanaJr — CHANGES_REQUESTED** [review]
Tackle the comments I provided. Also do a passing test even if it's a mockup

**Review by Habinezajanvier — APPROVED** [review]
LGTM

**Review by GasanaJr — APPROVED** [review]

**Pull request merged**